docs/root/version_history/current.rst

```diff
@@ -17,6 +17,7 @@ Minor Behavior Changes
 * http: fixed the 100-continue response path to properly handle upstream failure by sending 5xx responses. This behavior can be temporarily reverted by setting `envoy.reloadable_features.allow_500_after_100` to false.
 * http: the per-stream FilterState maintained by the HTTP connection manager will now provide read/write access to the downstream connection FilterState. As such, code that relies on interacting with this might
   see a change in behavior.
+* logger: change default log format to `"[%Y-%m-%d %T.%e][%t][%l][%n] [%g:%#] %v"` and default value of :option:`--log-format-prefix-with-location` to `0`.
```

Comment:
nit: logging to be consistent with the next line.

Also, just for my understanding, there will be no change in behavior for people that are not setting an explicit log format, right?

Reply:
Correct, there is no behavior change if `--log-format` is not explicitly used.